pixelate.py

```diff
@@ -287,12 +359,18 @@ def main(arguments):
     if selected_file:
         return start_matching_script(selected_file)
     #
-    logging.error("No (existing) file selected.")
+    toplevel = tkinter.Tk()
+    gui.InfoDialog(
```

Comment:
I don't like the gui here.

Reply:
Will change that back. Please also have a look at https://github.com/blackstream-x/pyxelate/discussions/113 – I am thinking of separating the install part from the pixelate.py script